code/modules/reqs/supply.dm

```diff
@@ -622,3 +622,43 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 	if(!SU)
 		SU = new(src)
 	return SU.interact(user)
+
+/obj/item/storage/backpack/marine/radiopack
+	name = "\improper TGMC radio operator backpack"
+	desc = "A backpack that resembles the ones old-age radio operator soldiers would use."
+	icon_state = "radiopack"
+	var/datum/supply_ui/requests/SU
+	var/datum/supply_beacon/beacon_datum
```

Comment:
both vars need autodoc

Reply:
I assume autodoc means a comment explaining what the var is. My coding lingo knowledge is very limited.